### ISSUE TITLE
Unable to access nested field data in a field group from a custom integration

### ISSUE THREAD

**Issue by steverowling** [verbb/formie#1901]
### Describe the bug

Similar to #1849, only now the problem I have now is with accessing sub-fields of a compound field (e.g. the Formie `Name` field) when it is located within a `Group` field.

I have a `Group` field called `inMemory`, which has a `Name` field in it called `inMemoryName`. When I set up a mapping in a custom integration to one of the sub-fields of the `Name` field, e.g. the `lastName` or `firstName` fields, I get the following error when the integrations `getFieldMappingValues()` function is run:

```
verbb\formie\base\Integration::getMappedFieldValue] Error trying to fetch mapped field value: “Invalid field handle: inMemoryName” /var/www/html/vendor/verbb/formie/src/elements/CustomElement.php:205
```

### Steps to reproduce

1. Create a new form with a compound field (e.g. a `Name` field) in a `Group` field.
2. Create an integration that maps a field to a sub-field of the compound field.

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes

### Craft CMS version

Craft Pro 5.1.5

### Plugin version

3.0.0-beta.11

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.0-beta.11"`.

**Comment by engram-design** [comment]
Fixed in [3.0.0-beta.12](https://github.com/verbb/formie/releases/tag/3.0.0-beta.12)
